scripts/jenkins/ardana/ansible/roles/ardana_deploy/tasks/run_lint.yml

```diff
@@ -0,0 +1,51 @@
+#
+# (c) Copyright 2018 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+- block:
+    - name: Create a temporary directory
+      command: mktemp -d
+      register: _temp_path
+
+    - name: Make a copy of "{{ lint_path }}"
+      synchronize:
+        src: '{{ lint_path }}'
+        dest: '{{ _temp_path.stdout }}'
+      delegate_to: "{{ inventory_hostname }}"
+
+    - name: Find ansible files under "{{ _temp_path.stdout }}"
+      command: >
+        find "{{ _temp_path.stdout }}" -type f
```

Comment:
@rtamalin does this mean that we won't be running the ansible linter on any of the other playbooks (keystone, cinder, swift, designate, neutron and so on) ?

Reply:
My understanding is that all of the various playbooks are combined into the scratch area, so when we run this on the scratch area we will lint all of them at the same time. But I'm not sure if that's truly all playbooks, or if there is a filter and only enabled playbooks are installed (in which case we may need to run different test scenarios that enable different combinations of playbooks, or rethink when and where we are linting).